scripts/int_local.sh

```diff
@@ -16,4 +16,4 @@ export MCLI_PUBLIC_API_KEY="${public_key}"
 export MCLI_PRIVATE_API_KEY="${private_key}"
 export MCLI_ORG_ID="${org_id}"
 
-AKO_INT_TEST=1 ginkgo --race --label-filter="${label}" --timeout 80m -v ./test/int -coverprofile cover.out
\ No newline at end of file
+AKO_INT_TEST=1 ginkgo --race --label-filter="${label}" --timeout 80m -v ./test/int/... -coverprofile cover.out
```

Comment:
Expanding all packages might be problematic once the go files inside `clusterwide` folder define the same package `int` on its parent. Tests will collapse with 2 test suites running at the same time.

Reply:
should I user a new line instead?
```suggestion
AKO_INT_TEST=1 ginkgo --race --label-filter="${label}" --timeout 80m -v ./test/int -coverprofile cover.out
AKO_INT_TEST=1 ginkgo --race --label-filter="${label}" --timeout 80m -v ./test/int/clusterwide... -coverprofile cover.out

```

Reply:
Done